src/table/virtualizedDataTable.jsx

```diff
@@ -932,242 +1329,153 @@ class VirtualizedDataTable extends Component {
-      },
-      table: {
-        WebkitUserSelect: 'none',
-        MozUserSelect: 'none',
-        msUserSelect: 'none',
-        userSelect: 'none',
-      },
-      tableAllowSelect: {
-      },
-      headerTable: {
-        overflow: 'hidden',
-        border: 'none',
-        WebkitUserSelect: 'none',
-        MozUserSelect: 'none',
-        msUserSelect: 'none',
-        userSelect: 'none',
-      },
-      tableHeader: {
-        fontSize: 16,
-        borderBottom: '1px solid #d3d3d3',
-        backgroundImage: 'linear-gradient(#fff,#efefef)',
-        padding: 0,
-        margin: 0,
-      },
-      tableGrid: {
-        fontSize: 16,
-      },
-      headerDiv: {
-        border: '1px solid #d3d3d3',
-        borderBottom: 'none',
-        overflowX: 'hidden',
-        overflowY: 'scroll',
-      },
-      tableDiv: {
-        border: '1px solid #d3d3d3',
-      },
-      headerCell: {
-        borderRight: '1px solid #d3d3d3',
-        display: 'flex',
-        alignItems: 'center',
-        flexDirection: 'row',
-        fontWeight: 700,
-        backgroundImage: 'linear-gradient(#fff, #efefef)',
-        height: '100%',
-      },
-      tableCell: {
-        borderRight: '1px solid #d3d3d3',
-        display: 'flex',
-        alignItems: 'center',
-        flexDirection: 'row',
-        height: '100%',
-      },
-      evenRows: {
-        background: 'white',
-      },
-      oddRows: {
-        background: '#f6f7f8',
-      },
-      resizer: {
-        cursor: 'ew-resize',
-        position: 'relative',
-        width: 4,
-        left: 'calc(100% - 4px)',
-      },
-      column: {
-        margin: 0,
-        paddingLeft: 10,
-      },
-      lastRow: {
-        borderBottom: '1px solid #d3d3d3',
-      },
-      tallDragLine: {
-        cursor: 'ew-resize',
-        position: 'absolute',
-        marginLeft: 1,
-        width: 2,
-        backgroundColor: 'blue',
-        zIndex: 9999999,
-      },
-      groupHeader: {
-        display: 'flex',
-        flexDirection: 'row',
-      },
-      groupOuter: {
-        display: 'flex',
-        flexDirection: 'row',
-        alignItems: 'center',
-        fontSize: 16,
-        fontWeight: 'bold',
-        paddingLeft: 16,
-        paddingRight: 16,
-        flexGrow: 0,
-        flexShrink: 0,
-        background: 'linear-gradient(#fff,#efefef)',
-        border: '1px solid #d3d3d3',
-        borderBottom: 'none',
-      },
-      hiddenGroupOuter: {
-        background: 'transparent',
-        border: 'none',
-      },
-      groupFarOuter: {
-        overflow: 'hidden',
-      },
+  prvRecomputeTables = () => {
+    const recomputeTable = (grid, forceOverflowInitial) => {
+      if (grid) {
+        grid.recomputeGridSize();
+        if (forceOverflowInitial) {
+          // eslint-disable-next-line react/no-find-dom-node
+          const domGrid = ReactDOM.findDOMNode(grid);
+          if (domGrid) {
+            domGrid.style.overflow = 'initial';
```

Comment:
HACK:  This styling is needed to remove scrollbars in the frozen part of the table.   Unfortunately, react-virtualized reapplies this, so we need to reapply it also.  Ugly, but there is no great solution without PR against react-virtualized.